keyboards/planck/keymaps/gunp/config.h

```diff
@@ -1,7 +1,7 @@
 #ifndef CONFIG_USER_H
```

Comment:
If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch? 

```suggestion
#pragma once
```